### DISCUSSION TITLE
New Properties and Methods for controls

### DISCUSSION THREAD

**Discussion by dmrvb** [twinbasic/twinbasic#1412] [Q&A]
Looking at the list of properties and methods for controls I see tB has many more than in VB.
Some, like mousecapture, look familar to those who have done some API, but others look like they are only of use to the extreme/hardcore users.
Should these be separated out in some way to keep only the common stuff visible to most users?
Does anybody have any documentaion for these yet?
As an example, what is CreateDeferredDispatchWrapper and does anybody use it? its so obscure that even Google can't tell me what it is.

**Comment by WaynePhillipsEA** [comment]
No need to be concerned by these.  They will be disappearing soon.  They are things used internally in base classes, currently exposed automatically due to the Implements-Via inheritance.
